src/landingPage/ToolsSection.module.scss

```diff
@@ -11,51 +13,22 @@
     margin: 0 auto;
     padding: 0 10vw;
     padding-bottom: 15vh;
-    display: flex;
-    flex-wrap: wrap;
-    justify-content: space-between;
 
-    h1 {
-      color: #e3e3e3;
-      font-size: 6rem;
-      font-weight: 900;
-      padding-top: 10vh;
-      padding-bottom: 8vh;
-      line-height: 100%;
-      width: 100%;
-  
-      .gradient {
-        background-clip: unset;
-        -webkit-background-clip: text;
-        -webkit-text-fill-color: transparent;
-        background-image: var(--std-gradient);
-        padding-bottom: 100%;
-      }
+    .titleBold {
+      padding-bottom: 100%;
```

Comment:
fixed conflict with main branch

Reply:
Removed this extra padding. I don't think it's needed?